src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -305,6 +305,9 @@
   - name: --attach-acr
     type: string
     short-summary: Grant the 'acrpull' role assignment to the ACR specified by name or resource ID.
+  - name: --api-server-authorized-ip-ranges
+    type: string
+    short-summary: Comma seperated list of authorized apiserver IP ranges. If left unset, all external traffic will be allowed.
```

Comment:
Lets update this to be samilar to L358, except the "" flag. ie.
Comma seperated list of authorized apiserver IP ranges. Set to 0.0.0.0/32 to restrict apiserver traffic to node pools.

Reply:
Sure - done